tests/wpunit/Tribe/functions/utilsTest.php

```diff
@@ -218,4 +218,24 @@ public function tribe_sanitize_deep_data_set() {
 	public function test_tribe_sanitize_deep( $input, $expected ) {
 		$this->assertEquals( $expected, tribe_sanitize_deep( $input ) );
 	}
-}
\ No newline at end of file
+
+	public function tribe_get_query_var_data_set() {
+		return [
+			'empty'                 => [ '', 'test', null ],
+			'not_a_url'             => [ 'foo-bar-baz', 'test', null ],
+			'abs_no_query'          => [ 'http://example.com/', 'test', null ],
+			'rel_no_query'          => [ '/index.php', 'test', null ],
+			'abs_query_arg_not_set' => [ 'http://example.com/?foo=bar', 'test', null ],
+			'rel_query_arg_not_set' => [ '/index.php?foo=bar', 'test', null ],
+			'abs_query_arg_set'     => [ 'http://example.com/?test=bar', 'test', 'bar' ],
+			'rel_query_arg_set'     => [ '/index.php?test=bar', 'test', 'bar' ],
```

Comment:
We should add a test to passing an array in the query args.

Reply:
Do we want to support arrays?

Reply:
Done